internal/test/wmcb/wmcb_test.go

```diff
@@ -221,24 +223,25 @@ func (vm *wmcbVM) handleHybridOverlay(nodeName string) error {
 		return nil
 	}
 
-	// Wait until the node object has the hybrid overlay subnet annotation. Otherwise the hybrid-overlay-node will fail to
-	// start
-	if err = waitForHybridOverlayAnnotation(nodeName); err != nil {
-		return fmt.Errorf("error waiting for hybrid overlay node annotation: %v", err)
+	// Wait until the node object has the hybrid overlay subnet annotation. Otherwise the hybrid-overlay-node will fail
```

Comment:
You need to fix these lines:
```
	// Check if the hybrid-overlay-node is running
	output, err := vm.Run("Get-Process -Name \"hybrid-overlay-node\"", true)
```
to check if the service is running.

Reply:
I think thats a potential failure point
running hybrid overlay causes the SSH connection to be broken, so based on timing, that check will fail if done while the network interface is being reset
It could be moved to after the wait period, but at that point we can just check for the network creation/ MAC address and get the same confirmation that hybrid-overlay has done its job.

Reply:
How is that a potential failure point? This is the first thing we do in handleHybridOverlay(). IOW, hybrid-overlay has not been run yet.

Reply:
My mistake, was confusing that with the waitForHybridOverlayToRun function.

Its a lot simpler to just check that the process is running, instead of parsing the output of sc.exe qc

Reply:
You don't need to use `sc.exe` but use `Get-Service -Name kubelet | Where-Object {$_.Status -eq "Running"}` or something similar. But I am fine leaving it as is.

Reply:
That command would need to be modified to throw an error if the list is empty, i'd prefer to just leave it as it is for the sake of simplicity